javaharness/java/arcs/android/BUILD

```diff
@@ -23,9 +23,9 @@ android_library(
     deps = [
         "//javaharness:autovalue",
         "//javaharness/java/arcs/api:api-android",
-        "@com_google_dagger",
-        "@flogger//jar",
+        "//third_party/java/dagger",
+        "//third_party/java/flogger",
         "@flogger_system_backend//jar",
```

Comment:
Can this be combined into a single jvm_library with the main flogger lib?

Reply:
Right now, it's deleted by copybara since the G3 target is hidden from visibility. Ideally we want Android Backend, but that's not released yet.  Let me see if there's a way to avoid keeping this.

Reply:
Ok, I got rid of this by making it a runtime_deps of flogger, so it is hidden behind the //third_party/java/flogger target. Since no src depends directly on the backend APIs and its needed only at runtime, it should be fine.